protocols/noise/src/io.rs

```diff
@@ -443,7 +414,7 @@ fn read_frame_len<R: AsyncRead + Unpin>(
     cx: &mut Context<'_>,
     buf: &mut [u8; 2],
     off: &mut usize,
-) -> Poll<Result<Option<u16>, std::io::Error>> {
+) -> Poll<io::Result<Option<u16>>> {
```

Comment:
Commenting on this PR as well, because I comment every time someone does that: I'm not a fan of doing random codestyle changes, especially for code that isn't otherwise touched by the PR.